charts/kueue/README.md

```diff
@@ -48,6 +48,46 @@ NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
 kueue-controller-manager       1/1     1            1           7s
 ```
 
+##### Cert Manager
```

Comment:
I prefer to keep the installation guide short and clean and focused on installation, not optional configuration. I suggest to move it to https://kueue.sigs.k8s.io/docs/tasks/manage/, probably under "Setup using cert-manager". Then describe how to do it for both webhooks and metrics.

Reply:
I think some short link to the tasks page from here is ok, but it seems more appropriate to have a proper page on setting up TLS. Also, cert-manager is for webhooks, not just metrics so this page would need to be expanded, and I think too long. We don't document other features here, just the installation steps and the params.

Reply:
So should I add a new task page for configuration installation?

The persona for ManageKueue is really the batch admistration and not the installation person.

Reply:
We don't distinguish these persons currently. Similarly, in the tasks for managing Kueue we instruct admins to install external frameworks such as MPIOperator, RayOperator, etc.

Reply:
Yes, I see it as a new tasks page under https://kueue.sigs.k8s.io/docs/tasks/manage/, some title ideas, might be "Setup TLS" or "Setup cert manager". I would envision the page to detail the setup for webhooks and metrics. Then, we can x-reference the pages for installation and the setup. IMO, but let me check with @tenzen-y ?

Reply:
I think we want to have 2 things

1. Add CertManager documentation to https://kueue.sigs.k8s.io/docs/tasks/manage/
2. Add a link to lead the detailed CertManger installation documentation to https://kueue.sigs.k8s.io/docs/installation/

Reply:
done! Let me know what you think.